hphp/util/CMakeLists.txt

```diff
@@ -71,6 +71,3 @@ endif()
 if (LIBNUMA_LIBRARIES)
   target_link_libraries(hphp_util ${LIBNUMA_LIBRARIES})
 endif()
-if (ENABLE_ASYNC_MYSQL)
-  add_dependencies(hphp_util fbmysqlclient)
-endif ()
```

Comment:
Also not sure why this was needed here, I don't see any references to mysql in hphp_util.